research/src/pages/select/select.proposal.mdx

```diff
@@ -116,11 +116,55 @@ children:
   Should this apply to the entire anatomy or solely the &lt;option&gt; and &lt;optgroup&gt; element
 </p>
 
+## Events <a class="link" href="#events" id="events"></a>
+
+### select
+
+| Event    | Behavior                                                                                         | Impacts          |
+| -------- | ------------------------------------------------------------------------------------------------ | ---------------- |
+| `change` | Updates the textContent property of the selected-value slot with the value property text content | textContent prop |
+
+### part button
+
+| Event            | Behavior                           | Impacts                                                                  |
+| ---------------- | ---------------------------------- | ------------------------------------------------------------------------ |
+| `click`          | Toggles the open state on the host | [open](#open-state) state                                                |
+| `click`          | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `pointerup`      | Toggles the open state on the host | [open](#open-state) state                                                |
```

Comment:
Having both `click` and `pointerup` in the table is confusing me a bit for two reasons:
1) If I mouse-click on an element both `click` and `pointerup` are fired.  This suggests to me that both the `click` and the `pointerup` rows of the table are "fired", meaning that everything toggles twice and nothing really happens (I might just be taking this document more literally than is intended, but it would be nice to clarify this somehow).
2) `pointerup` doesn't seem like the right choice because of the following behavior: If I click and hold the mouse button down somewhere else on the page, drag the mouse over the button part, and then release the mouse, I'll get a `pointerup` -- as opposed to `click` which also requires the mouse-down to have been on the same element.  This seems wrong.  The native `<select>` in Chrome on Windows and Mac opens on mousedown, and `<input type="button">` is usually used with `click`.  It seems to me like we should stick with one of those unless there is some compelling reason I missed.